### ISSUE TITLE
Set time of life for chats

### ISSUE THREAD

**Issue by ankar84** [RocketChat/feature-requests#573]
**Is your feature request related to a problem? Please describe.**
Hundreds of stale chats consumes user space of sidebar and enlarge Rocket.Chat database with no reason.

**Describe the solution you'd like**
Nice solution for that could be a setting on creation of chat process when user choose how long that chat will remain on server:
- Never deleted
- Delete after and dropdown list with predefined values like day, week, month
- Delete on with ability to pick date from calendar

**Describe alternatives you've considered**
I don't know

**Additional context**
That feature request a bit related to https://github.com/RocketChat/feature-requests/issues/402 i\in terms of clearing server of stale and unneeded information like chats.

**Comment by krenuser** [comment]
https://[your.rocket.chat.domain]/admin/settings/RetentionPolicy

Isn't that what you've wanted?

![image](https://user-images.githubusercontent.com/32645945/182583945-4b854eb9-fcaf-4295-9b50-1b62cde8e1be.png)

**Comment by ankar84** [comment]
> Isn't that what you've wanted?

No. We use Retention Policy a lot. And actually because of Retention Policy we have a lot of empty (with totally cleaned messages) chats that we want to be deleted automatically.
